lib/taskana-core/src/test/java/acceptance/report/ProvideWorkbasketPriorityReportAccTest.java

```diff
@@ -329,4 +336,173 @@ void should_ReturnItemsOfWorkbasketReport_When_FilteringWithAllCustomFieldsNotIn
     int[] row1 = report.getRow("USER-1-1").getCells();
     assertThat(row1).isEqualTo(new int[] {1, 0, 0});
   }
+
+  @WithAccessId(user = "monitor")
+  @TestFactory
+  Stream<DynamicTest> should_ApplyFilter_When_QueryingForCustomIntIn() {
+    List<Pair<TaskCustomIntField, Integer>> testCases =
+        List.of(
+            Pair.of(TaskCustomIntField.CUSTOM_INT_1, 1),
+            Pair.of(TaskCustomIntField.CUSTOM_INT_2, 2),
+            Pair.of(TaskCustomIntField.CUSTOM_INT_3, 3),
+            Pair.of(TaskCustomIntField.CUSTOM_INT_4, 4),
+            Pair.of(TaskCustomIntField.CUSTOM_INT_5, 5),
+            Pair.of(TaskCustomIntField.CUSTOM_INT_6, 6),
+            Pair.of(TaskCustomIntField.CUSTOM_INT_7, 7),
+            Pair.of(TaskCustomIntField.CUSTOM_INT_8, 8));
+
+    ThrowingConsumer<Pair<TaskCustomIntField, Integer>> test =
+        p -> {
+          WorkbasketPriorityReport report =
+              MONITOR_SERVICE
+                  .createWorkbasketPriorityReportBuilder()
+                  .withColumnHeaders(DEFAULT_TEST_HEADERS)
+                  .customIntAttributeIn(p.getLeft(), p.getRight())
+                  .inWorkingDays()
+                  .buildReport();
+          assertThat(report).isNotNull();
+          assertThat(report.rowSize()).isEqualTo(5);
+          assertThat(report.getRow("USER-1-1").getCells()).isEqualTo(new int[] {20, 0, 0});
+        };
+
+    return DynamicTest.stream(testCases.iterator(), p -> p.getLeft().name(), test);
```

Comment:
```suggestion
    return DynamicTest.stream(testCases.iterator(), Pair::getLeft, test);
```

I am not sure, does this work?

Reply:
No, it doesn't compile